README.md

```diff
@@ -2,7 +2,7 @@
 
 Shameless cheap knock off version of Shopify internal developer tool `dev`.
 
-Command line utility `fk` (stands for *F*antastic *K*it)
+Command line utility `fk` (stands for **F**antastic **K**it)
```

Comment:
`fk` sounds like `f*ck` tbh :stuck_out_tongue:

Reply:
I think that's point.

Reply:
I have thought about replacing `fk <cmd>` with `kit <cmd>`, but I guess until this project is being used by alot of people, we can just have some fun with the names for now :stuck_out_tongue: